container/Dockerfile.rhel7

```diff
@@ -0,0 +1,67 @@
+FROM centos:7.5.1804
```

Comment:
In theory with correct merge you should not have this file int the PR.

Reply:
It came from original repo, so I left it, meaning  I have original added manually